public/app/features/dashboard/panel_editor/HelpToggle.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from '@grafana/ui';
+import { css, cx } from 'emotion';
+import React, { useState } from 'react';
+
+const helpStyle = css`
+  margin-top: 4px;
+`;
+
+export const HelpToggle: React.FunctionComponent = ({ children }) => {
+  const [isHelpVisible, setIsHelpVisible] = useState(false);
+
+  return (
+    <>
+      <label className="gf-form-label query-keyword pointer" onClick={_ => setIsHelpVisible(!isHelpVisible)}>
```

Comment:
nit: would it be possible to skip the dependency on the legacy classNames and use emotion instead? Maybe replace this label with components from grafana/ui that embeds the styling?

Reply:
Good idea, will do

Reply:
Was thinking about this and I think `<button>` might be the more semantically appropriate tag, though I'd like to leave the styling as-is so it fits in well with query editor styling. Might have to factor out some of the `<Label>` styles.